docs/Configuration.rst

```diff
@@ -5981,7 +6036,166 @@ data is stored in the database.
+
+     -  A second password that can be configured in properties:
+
+    .. code-block:: jproperties
+
+          exo.wallet.admin.key=changeThisKey
+
+
+    .. warning:: The password can't be changed once the platform is started for the first time. In fact, this password will be used to encrypt the *Admin wallet*'s private key that will be stored in database.
+        If its value is modified after server startup, the private key of admin wallet won't be decrypted anymore, preventing all administrative operations.
+
+
+  The *Admin wallet* can be used by */platform/rewarding* group members to initialize other wallets.
+  The private key of the admin wallet cannot be accessed by any user to avoid exposing its funds to an unauthorized person.
+
+* **User wallet**
+
+  By default, any user of the platform can create his own wallet. A property is added to restrict access to wallet application to a group of users:
+
+    .. code-block:: jproperties
+
+          # Example of expressions
+          
+          # Any user
+          exo.wallet.accessPermission=
+          # All connected users of the platform
+          exo.wallet.accessPermission=/platform/users
+          # All administrators of the platform
+          exo.wallet.accessPermission=*:/platform/administrators
+          # All members of space associated to group internal_space
+          exo.wallet.accessPermission=member:/spaces/internal_space
+
+* **Space wallet**
+
+  Space managers can add the wallet application in their space a wallet for a space using :ref:`Managing space applications <ManagingSpaceApplication>` UI:
+
+  |SpaceAddWallet|
+
+  Space wallets can be managed by one or multiple space manager. In fact, the space wallet is similar to user wallet. To use it, it must be activated by an admin. The operation is carried out by the *Admin Wallet* account.
+  By default, any space member can access to the space wallet in *readonly* mode.
+
+Token contract
+~~~~~~~~~~~~~~
+
+The contract address used for the main token currency can be configured using the following variable:
+
+    .. code-block:: jproperties
+
+          exo.wallet.blockchain.token.address=0xc76987d43b77c45d51653b6eb110b9174acce8fb
+
+This address is the official contract address for the official rewarding token. It shouldn't change through eXo Platform versions on the Ethereum Mainnet blockchain. This crypto-currency token, is shared by all eXo Community users.
+
+It is an `ERC-20 <https://eips.ethereum.org/EIPS/eip-20>`__ Token contract that is deployed on Ethereum Mainnet.
+
+To be able to receive some tokens, a wallet address must be initialized by a *token contract* administrator on the blockchain (this cannot be done on your eXo Platform server).
```

Comment:
```suggestion
To be able to receive some tokens, a wallet address must be initialized by a *token contract* administrator (eXo staff) on the blockchain (this cannot be done on your eXo Platform server). 
```

Reply:
Not really *eXo staff* only. In fact, we have to promote *Admin wallet*, of eXo Platform instances deployed in customers/community servers, as administrator with level 2 on servers to be able to reward and initialize wallets of other users of their server instance.